apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -69,8 +69,10 @@ export class WidgetsController {
     @SubscriberSession() subscriberSession: SubscriberEntity,
     @Query('page') page: number,
     @Query('feedIdentifier') feedId: string[] | string,
-    @Query('seen') seen: boolean | undefined = undefined
+    @Query('seen') seen?: string
```

Comment:
Can we override this only on the swagger implementation to show a boolean? Perhaps using the ApiQuery decorator here?